tests/controller/index.spec.ts

```diff
@@ -16,7 +16,9 @@ describe('Index Controller', () => {
 
 describe('Message Controller', () => {
   test("responds with a user'/s message", () => {
-    const mockRequest: Partial<Request<any>> = { params: { message: 'I am a user!' } }
+    const mockRequest: Partial<Request<any>> = {
+      params: { message: 'I am a user!' }
+    }
```

Comment:
This file and folder can be removed since the controller was removed from implementation!

Reply:
Removing the files/folder! :)